### ISSUE TITLE
Port existing programs over

### ISSUE THREAD

**Issue by adrian-goe** [Yes-Theory-Fam/yesbot-ts#263]
Port existing programs over (can be done incrementally, keeping the current event handlers intact and removing programs from the imperative handler once moved to the decorator based one)

**Create new issues for small packages**
we want to avoid a large pull request so that they can be reviewed more easily and also worked off in parallel.
So we like to open a new issue with a pull request for every single subprogram.

Issues don't cost us any money, so there is no need to save money 😉

**Comment by adrian-goe** [comment]
@findtravishere we have an kanban Board for the Refactoring here in GitHub under projects.
All issued resulted from this here will be found there as well as all other issues related to the refactoring 😊

**Comment by findtravishere** [comment]
i cant see the kanban board on mobile 👀 my bad

**Comment by geisterfurz007** [comment]
Created

#424 - #440 for porting the existing commands over and added them to the [Refactoring Project](https://github.com/Yes-Theory-Fam/yesbot-ts/projects/2#card-63256676).
